packages/twenty-front/src/modules/views/components/EditableFilterDropdownButton.tsx

```diff
@@ -68,6 +68,7 @@ export const EditableFilterDropdownButton = ({
 
   return (
     <Dropdown
+      dropdownMenuWidth={280}
```

Comment:
This should already be applied in `MultipleFiltersDropdownFilterOnFilterChangedEffect` could you take a look.
I believe it is because the dropdownId do not match: viewFilter.fieldMetadataId vs ObjectFilterDropdownId